src/issues/issue-2/david-byrne/index.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from "react"
+import Img from "gatsby-image"
+import { useStaticQuery, graphql } from "gatsby"
+import { useMediaQuery } from "react-responsive"
+
+import Header from "./Header"
+import AnimatedWords from "./AnimatedWords"
+import Pagination from "./Pagination"
+
+import useMedia from "./useMedia"
+import useMeasure from "./useMeasure"
+import usePositions from "./usePositions"
+import albumByYearData from "./albumsByYear.js"
+import { mapImagesToAlbums, calculateGridItems } from "./utility"
+import {
+  Background,
+  Text,
+  FlexContainer,
+  ImgWrap,
+  WordCountContainer,
+} from "./styled"
+import { breakpoints } from "../../../styles/layout"
+
+const xOffset = window.innerWidth / 2
+
+const DavidByrne = () => {
+  const isPhoneWide = useMediaQuery({
+    query: breakpoints.phoneWide,
+  })
+  // Query images
+  const { mini, big } = useStaticQuery(graphql`
+    query {
+      mini: allFile(
+        filter: { relativeDirectory: { eq: "issue2-images/david-byrne/mini" } }
+      ) {
+        ...ImageFragment
+      }
+      big: allFile(
+        filter: { relativeDirectory: { eq: "issue2-images/david-byrne/big" } }
+      ) {
+        ...ImageFragment
+      }
+    }
+  `)
+  // Map images to album data
+  const albums = mapImagesToAlbums(albumByYearData, big, mini)
```

Comment:
since I'm fetching the images from graphql and fetching the album data separately, I need this utility function to combine them into one data structure

Reply:
Is this using a promise?